pkg/minikube/bootstrapper/bsutil/testdata/v1.17/containerd-pod-network-cidr.yaml

```diff
@@ -35,11 +36,3 @@ networking:
   dnsDomain: cluster.local
   podSubnet: "192.168.32.0/20"
   serviceSubnet: 10.96.0.0/12
----
```

Comment:
Is this expected? Shouldn't we still be setting the eviction settings for this version?